core/src/main/java/org/infinispan/manager/EmbeddedCacheManager.java

```diff
@@ -373,4 +374,10 @@ default EmbeddedCacheManagerAdmin administration() {
    }
 
    ClassWhiteList getClassWhiteList();
+
+   /**
+    * @since 11.0
+    * @return the {@link EncoderRegistry} associated with the cache manager.
+    */
+   EncoderRegistry getEncoderRegistry();
```

Comment:
Should this really be public API?

Reply:
What API do you suggest to expose that a conversion is supported and to convert data? ATM, at least internally, it grabs this EncoderRegistry from the registry every time

Reply:
Why would an application need to ask at runtime whether a conversion is supported? Internal components can keep injecting the `EncoderRegistry` from the global component registry.

Reply:
> Why would an application need to ask at runtime whether a conversion is supported? 

Those methods were in the DataConversion, and this PR deprecated them. Are you proposing to just deprecate them and provide no alternative?

Reply:
Yes, if someone actually needs an alternative we will provide it, but the deprecated methods will still work until the affected users realize they are affected and ask for an alternative.

Reply:
Ok, PR updated